python/java-stubs/util/__init__.pyi

```diff
@@ -0,0 +1,3543 @@
+from jpype._jcollection import _JCollection
```

Comment:
have you considered not versioning these stubs, so leaving them to be generated on build/release? it'd be nice if we can do that, so to keep the repo cleaner

Reply:
It's a possibility but makes it harder for a maintainer that eventually wants to improve only the python part of the project, because every new collaborator will need to generate the stubs.